app/src/main/java/net/skyscanner/backpack/demo/data/ComponentRegistry.kt

```diff
@@ -86,6 +86,10 @@ object ComponentRegistry {
         "Icon alignment" story NodeData { Story of R.layout.fragment_button_icon_alignment },
         "Icon alignment with RTL" story NodeData { Story of R.layout.fragment_button_icon_alignment_rtl }
       )),
+    "ButtonLink" story NodeData({ children -> SubStory of children },
```

Comment:
Let's redirect it to R.layout.fragment_button_link for now. No need to create additional screen yet.

Reply:
I've added one for icon alignment